app/src/main/java/com/github/multimatum_team/multimatum/util/LocalDateTimeSerializer.kt

```diff
@@ -0,0 +1,24 @@
+package com.github.multimatum_team.multimatum.util
+
+import com.google.gson.JsonElement
+import com.google.gson.JsonPrimitive
+import com.google.gson.JsonSerializationContext
+import com.google.gson.JsonSerializer
+import java.lang.reflect.Type
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+/**
+ * custom serializer to serialize LocalDateTime into Json with Gson
+ * took from https://www.javaguides.net/2019/11/gson-localdatetime-localdate.html
+ */
+internal class LocalDateTimeSerializer : JsonSerializer<LocalDateTime?> {
+    override fun serialize(localDateTime: LocalDateTime?, srcType: Type?, context: JsonSerializationContext?):
+            JsonElement {
+        return JsonPrimitive(formatter.format(localDateTime))
+    }
+
+    companion object {
+        private val formatter = DateTimeFormatter.ofPattern("d::MM::uuuu HH::mm::ss")
```

Comment:
what if day is more than 9, we'ld need 2 char "dd"

Reply:
Good catch, I think it would be better to use a standard date format too.

Reply:
the fact that there is only one d does not mean that it takes at most one digit. This is a standard format. This is just mean that if the d is less than 10 for example 9. it will be formatted as just 9 instead of 09 like with dd.